scripts/combine/setupCombine.py

```diff
@@ -242,15 +268,18 @@ def setup(args, inputFile, fitvar, xnorm=False):
         
     passSystToFakes = wmass and not args.skipSignalSystOnFakes and args.qcdProcessName not in excludeGroup and (filterGroup == None or args.qcdProcessName in filterGroup) and not xnorm
 
-    cardTool.addProcessGroup("single_v_samples", lambda x: x[0] in ["W", "Z"] and x[1] not in ["W","Z"])
+    # FIXME: try to reuse some lambda functions to avoid mistakes, or maybe define some general functions?
+    cardTool.addProcessGroup("single_v_samples", lambda x: any(x.startswith(init) for init in ["W", "Z", "BkgW", "BkgZ"]) and x[1] not in ["W","Z"])
     if wmass:
-        cardTool.addProcessGroup("w_samples", lambda x: x[0] == "W" and x[1] not in ["W","Z"])
+        cardTool.addProcessGroup("w_samples", lambda x: any(x.startswith(init) for init in ["W", "BkgW"]) and x[1] not in ["W","Z"])
```

Comment:
Could make the same function, with an arg for boson = ["W"], ["Z"], ["W", "Z"] or similar

Reply:
Can think of it, what this is doing here is selecting single W samples excluding WW and ZZ and WZ

Reply:
So I added something, eventually the function may be moved elsewhere and modified as needed, for now it is just a preliminary version